tools/travis/setup.sh

```diff
@@ -34,7 +34,7 @@ function retry() {
 pip install --user couchdb
 
 # Ansible
-pip install --user ansible==2.5.2
+pip install --user ansible==2.8.18
```

Comment:
ansible 2.5.2 is not compatible with python3, and some of our ansible script doesn't compatible ansible versions beyond 2.8

Reply:
So do we need to migrate all scripts?
If so, it might be easier to fix the `flake8` issue, for now, to normalize CI tests and incrementally introduce python3.

Reply:
I found just one error while using ansible2.10, which is in the `cli/deploy.yml#name: "Download release archive to build directory ..."`

the `headers` option for `get_url` should use map instead of string